exec/apk-gen.sh

```diff
@@ -39,12 +40,14 @@ then
 
 	git push origin apk --force --quiet > /dev/null
 
-	curl https://$APPETIZE_API_TOKEN@api.appetize.io/v1/apps/mbpprq4xj92c119j7nxdhttjm0 -H 'Content-Type: application/json' -d '{"url":"https://github.com/fossasia/susi_android/raw/apk/app-debug.apk", "note": "Update SUSI Preview"}'
+	curl https://$APPETIZE_API_TOKEN@api.appetize.io/v1/apps/mbpprq4xj92c119j7nxdhttjm0 -H 'Content-Type: application/json' -d '{"url":"https://github.com/fossasia/susi_android/raw/apk/app-fdroid-debug.apk", "note": "Update SUSI Preview"}'
+	curl https://$APPETIZE_API_TOKEN@api.appetize.io/v1/apps/mbpprq4xj92c119j7nxdhttjm0 -H 'Content-Type: application/json' -d '{"url":"https://github.com/fossasia/susi_android/raw/apk/app-playStore-debug.apk", "note": "Update SUSI Preview"}'
 
 	# Publish App to Play Store
 	if [ "$CIRCLE_BRANCH" == "$PUBLISH_BRANCH" ]; then
 		echo "Publishing app to Play Store"
 		gem install fastlane
-		fastlane supply --apk app-release.apk --track alpha --json_key ../exec/fastlane.json --package_name $PACKAGE_NAME
+		fastlane supply --apk app-fdroid-release.apk --track alpha --json_key ../exec/fastlane.json --package_name $PACKAGE_NAME
```

Comment:
Why are we uploading fdroid version to Play Store and also why would it even work?